web_frontend/package.json

```diff
@@ -37,10 +40,9 @@
     "@types/regenerator-runtime": "^0.13.1",
     "@typescript-eslint/eslint-plugin": "^6.0.0",
     "@typescript-eslint/parser": "^6.0.0",
-    "@vitejs/plugin-react-swc": "^3.3.2",
+    "@vitejs/plugin-react": "^4.2.1",
```

Comment:
I think we were using SWC for faster builds, are you able to get it to work using SWC?